File: Java Course/Basics/anagram.java
import java.util.Scanner;

public class anagram {
    public static void main(String[] args) {
        //  anagram
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the word 1 : ");
        String word1 = input.nextLine();
        System.out.print("Enter the word 2 : ");
        String word2 = input.nextLine();
        int count = 0;
        if(word1.length() == word2.length()){
            for(int i=0; i<word1.length(); i++){
                for(int j=0; j<word2.length(); j++){
                    if(word1.charAt(i) == word2.charAt(j)){
                        count++;
                        break;
                    }
                }
            }
        }
        if(count == word1.length()){
            System.out.println("Anagram");
        }
        else{
            System.out.println("not Anagram");
        }
        
        input.close();
    }
}

File: Python Course/Problems/anagram.py
# anagram
word1 = input("Enter the word 1 : ")
word2 = input("Enter the word 2 : ")
arr1 = list(word1)
arr2 = list(word2)
count = 0
if len(arr1) == len(arr2):
    for i in range(len(arr1)):
        for j in range(len(arr2)):
            if (arr1[i]==arr2[j]):
                count+=1
                break
if count == len(arr1):
    print("It is anagram")
else:
    print("It is not anagram")